app/gui/src/ide.rs

```diff
@@ -102,6 +108,7 @@ pub struct FailedIde {
 }
 
 
+
```

Comment:
I think we should have only one empty line here.

Reply:
Ah, that must have been an accident. Sorry!